src/components/Person/Person.jsx

```diff
@@ -1 +1,33 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+
+      {age && (
+        <p className="Person__age">
+          {`I am ${age}`}
+        </p>
+      )}
+
+      <p className="Person__partner">
+        {isMarried ? (
+          `${partnerName} is my ${sex === 'm' ? 'wife' : 'husband'}`
+        ) : (
+          'I am not married'
```

Comment:
Try to avoid nested ternary,  you can create a separate variable for a partner and use it here

`const partner = sex === 'm' ? 'wife' : 'husband;`